mobile/src/core/store/app/reducer.ts

```diff
@@ -8,8 +8,8 @@ export interface IAppReduxState {
 export const appState = (): IAppReduxState => {
   return {
     isAppLoaded: false
-  }
-}
+  };
+};
 
 export const app: Reducer = (
```

Comment:
once this tends to grow, we can take leverage of JSON Object Mapping to get rid of those switch statements...